Server/Plugins/APIDump/InfoFile.html

```diff
@@ -217,6 +225,24 @@
 
 
 			<hr />
+			<a name="Categories"><h2>Categories</h2></a>
+
+			<p>The optional Categories table provides descriptions for categories in the generated documentation. Commands can have categories with or without category descriptions in this table. The documentation generator will output a table of listed categories along with their description.</p>
```

Comment:
This is a somewhat twisted description, I had to read it three times to understand it (and I know how all this works).

How about: 
```
The optional Categories table provides descriptions for command categories in the generated
documentation. The documentation generator will group the commands by their specified
Category ("General" by default) and each category will have the specified description
written to it.
```